ts/src/test/base/tests.init.ts

```diff
@@ -2,16 +2,17 @@
 /* eslint-disable */
 import ololog from 'ololog'
 import { strictEqual, deepEqual } from 'assert' // for easier debugging
-import { functions } from '../../../ccxt.js'
+import { Exchange, functions } from '../../../ccxt.js'
```

Comment:
right now these file is a bit weird, contains tests, imports tests and run tests that are transpiled and language-specific.

i think I would, remove the tests from the file and move them to a tests file like test.uncamelcase.ts

then I would make the tests imports and the `baseTestsInit` transpilable.

Regarding the language-specific functions I think i would put them on a different file inside a method like `languageSpecificMethods` and here we would call that method as well (the implementation in the other languages can be a empty method)

Reply:
@ttodua something like this

```Typescript
// language-specific imports, we don't touch them
import ololog from 'ololog'

// tests imports, added programatically in py/php/cs
import testTypes from './test.type.js'
import testNumber from './test.number.js'
import testDatetime from './test.datetime.js'
import testCrypto from './test.crypto.js'
import testExtend from './test.extend.js';


// this functions gets transpiled
function baseTestsInit (exchange) {
    testLanguageSpecific (); // this method can be empty in py/php/cs
    testExtend ();
    testUnCamelCase ();
```

Reply:
by doing that, everytime we add a method we don't need to import it manually in py/php/cs

Reply:
yes, my final goal was to make them transpilable too, just for this stage i didnt go that further, because the main reason i refrained from more changes, was that in JS/PY/PHP we already had those base-method tests files more alike "language-specific" (so, those files didnt even export anything and just calling those files directly run tests), so at first I thought to place them in "base/tests.init.ext" which wont be transpilable at this stage, but soon after merger i would take care of that too, as again, this PR might not be perfect but because of too much changes, i didn't emphasize on these details.
wdyt, should I make those changes now? or you were going to do them? lmk and I do as you say

Reply:
eventually, after rolling around this PR, i've recoded those base parts and pushed as you said. Now:
1) `base/tests.init.ts` is transpilable, it does not contain any custom imports, but only needed imports (those imports are passed into transpiled files too)
2) i've placed `test.languageSpecific.ext` (non-transpilable) files in every lang (in c# i've just added an empty method in BaseTest partial class), so that `languageSpecific` tests can include custom things manually across langs.

Reply:
so is this comment addressed?